AppCenter/AppCenter/Internals/Channel/MSChannelUnitDefault.m

```diff
@@ -90,7 +90,7 @@ - (void)ingestionDidReceiveFatalError:(__unused id<MSIngestionProtocol>)ingestio
 
 #pragma mark - Managing queue
 
-- (void)enqueueItem:(id<MSLog>)item {
+- (void)enqueueItem:(id<MSLog>)item flags:(MSFlags)flags {
```

Comment:
I think it would also be useful to have a wrapper without the `flags` parameter here that always passes `flags:MSFlagsNone`

Reply:
This is not a public API though.